core/resources/scene/scene.yaml

```diff
@@ -16,6 +16,22 @@ textures:
     water-texture:
         url: normals.jpg
         filtering: nearest
+
+# provide global fontfile names to be used for this stylesheet
+# these could be bundled with the tangram repo or could be loaded from system path
+# priority is given to bundled fonts
+fonts:
+    font1:
+        name: Roboto
+        weight: Regular
+    font2:
+        name: FireSans
+        weight: Medium
+    font3:
+        name: Roboto
```

Comment:
Thats a good point and yes it can (will) be done. 

I will however wait for discussion on "typeface" font parameter to come to a conclusion and then implement the code using "name", "weight" and "face" parameters of a font.

So, as proposed earlier for style rule matching and inheritance and (native) implementation sakes, its much better to have the following:

```
draw:
        text:
              font:
                     size: 15px
                     name: Roboto
                     weight: Bold
                     face: Italics
```
instead of
```
draw:
        text:
              font:
                     typeface: Italics Bold 15px Roboto
```